lib/common/Memory/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+add_library (Chakra.Common.Memory
+    Allocator.cpp
+    ArenaAllocator.cpp
+    CommonMemoryPch.cpp
+    EtwMemoryTracking.cpp
+    ForcedMemoryConstraints.cpp
+    HeapAllocator.cpp
+    HeapAllocatorOperators.cpp
+
+    # xplat-todo: Fix me
+    #HeapBlock.cpp
+
+    HeapBlockMap.cpp
+    heapbucket.cpp
+
+    # xplat-todo: Fix me
+    #heapinfo.cpp
+
+    idleDecommitPageAllocator.cpp
```

Comment:
OSX has a case insensitive FS by default, so there may be more of these that I'm missing